spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-binder-functional-sample/spring-cloud-gcp-pubsub-stream-binder-functional-sample-test/pom.xml

```diff
@@ -48,5 +48,16 @@
 
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
Was there an issue with excluding this from the pom?

Reply:
Yeah - since this module only has a `src/test` folder, the `spring-boot-maven-plugin` complained it couldn't find a `Main` class.

Reply:
Gotcha I see.

Reply:
I pulled this one level up, because I think that build error was caused by the JAR not looking right after being repackaged.